index.js

```diff
@@ -16,6 +17,9 @@ async function importEsm(specifier, module) {
         } catch {
             resolvedPath = req.resolve(specifier);
         }
+        if(os.platform() === "win32") {
+            resolvedPath = 'file://' + resolvedPath
```

Comment:
Should this use `const url = require('url'); url.pathToFileURL(` instead?

Reply:
yeh, using `pathToFileURL` makes more sense as #7 did.